orchestrators/FinanceOperations/tests/Unit/Rules/BudgetAvailableRuleTest.php

```
@@ -4,474 +4,155 @@
+                private bool $active,
+                private string $availableAmount,
+                private bool $found,
+                private string $expectedTenantId,
+                private string $expectedBudgetId,
+                private ?string $expectedCostCenterId,
+            ) {}
+
+            public function getBudget(string $tenantId, string $budgetId): ?BudgetRuleViewInterface
+            {
+                if ($tenantId !== $this->expectedTenantId || $budgetId !== $this->expectedBudgetId) {
+                    throw new \InvalidArgumentException("Unexpected tenant or budget: got $tenantId/$budgetId, expected {$this->expectedTenantId}/{$this->expectedBudgetId}");
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '128', column '31'). (undefined)

(MissingImport)